torch/csrc/jit/codegen/cuda/shape_inference.h

```diff
@@ -8,6 +8,8 @@ namespace jit {
 namespace fuser {
 namespace cuda {
 
+std::vector<int64_t> getTensorSizes(TensorTypePtr const& tensor_type);
```

Comment:
nitpick: no need to expose this in header.

Reply:
I'm going to use this in `parser.cpp` for handling the view / reshape operation, but it isn't necessary for this PR.